homeassistant/components/mvglive/sensor.py

```diff
@@ -18,44 +19,31 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_NEXT_DEPARTURE = "nextdeparture"
-
 CONF_STATION = "station"
 CONF_DESTINATIONS = "destinations"
-CONF_DIRECTIONS = "directions"
 CONF_LINES = "lines"
 CONF_PRODUCTS = "products"
 CONF_TIMEOFFSET = "timeoffset"
 CONF_NUMBER = "number"
 
-DEFAULT_PRODUCT = ["U-Bahn", "Tram", "Bus", "ExpressBus", "S-Bahn", "Nachteule"]
+NONE_ICON = "mdi:clock"
+
+ATTRIBUTION = "Data provided by mvg.de"
 
-ICONS = {
-    "U-Bahn": "mdi:subway",
-    "Tram": "mdi:tram",
-    "Bus": "mdi:bus",
-    "ExpressBus": "mdi:bus",
-    "S-Bahn": "mdi:train",
-    "Nachteule": "mdi:owl",
-    "SEV": "mdi:checkbox-blank-circle-outline",
-    "-": "mdi:clock",
-}
-ATTRIBUTION = "Data provided by MVG-live.de"
+ENTITY_ID_FORMAT = "mvglive.{}"
 
-SCAN_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=45)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_NEXT_DEPARTURE): [
             {
                 vol.Required(CONF_STATION): cv.string,
                 vol.Optional(CONF_DESTINATIONS, default=[""]): cv.ensure_list_csv,
-                vol.Optional(CONF_DIRECTIONS, default=[""]): cv.ensure_list_csv,
```

Comment:
We are not allowed to change the schemas nor should we in this occassion.
Leave it and create an issue in `setup_platform` that the key needs to be removed

Reply:
Ok, so this means I have to add the line 24 back again as well?